code/src/App.js

```diff
@@ -1,9 +1,68 @@
-import React from 'react';
+/* eslint-disable */
+import React, { useState } from 'react';
+import QuestionText from 'Components/QuestionText.js';
+import QuestionRadio from 'Components/QuestionRadio.js';
+import Summary from 'Components/Summary.js';
+import Footer from 'Components/Footer.js';
+import Header from './Components/Header.js';
+import QuestionDropdownGenre from './Components/QuestionDropdownGenre';
+import QuestionDropdownNumber from './Components/QuestionDropdownNumber';
 
 export const App = () => {
+  const [genre, setGenre] = useState('')
+  const [number, setNumber] = useState('')
+  const [recommendation, setRecommendation] = useState('')
+  const [radio, setRadio] = useState('')
+  const [showSummary, setShowSummary] = useState(false);
+  const [showForm, setShowForm] = useState(true);
+
+  const handleSubmit = ((event) => {
+    event.preventDefault()
+    setShowSummary(true)
+    setShowForm(false)
+  })
+
+  const surveyAnswered = () => {
```

Comment:
How does this work? Is it true if the question is an empty string? I would guess the opposite but I don't understand this true /false thing.